examples/component_v3.1.0.yaml

```diff
@@ -0,0 +1,79 @@
+documentation_complete: false
+name: Amazon Elastic Compute Cloud
+references:
+- name: Reference
+  path: http://VerificationURL.com
+  type: URL
+satisfies:
+- control_key: CM-2
+  covered_by:
+  - verification_key: EC2_Verification_1
+  - component_key: UAA
+    system_key: CloudFoundry
+    verification_key: UAA_Verification_1
+  implementation_status: partial
```

Comment:
Remove this line...the user shouldn't have both `implementation_status` and `implementation_statuses`.